conf/messages.en

```diff
@@ -982,6 +982,25 @@ authorisedOfficialPreviousAddress.error.required = Select yes if their address h
 
 authorisedOfficialPreviousAddress.checkYourAnswersLabel = Address changed in the last 12 months
 
+# Authorised Official Previous Address Messages
```

Comment:
both this page and previous ('address changed in last 12 months?') use prefix `authorisedOfficialPreviousAddress`

Reply:
it is naming convetion conflicts between authorisedOfficialPreviousAddress and isAuthorisedOfficialPreviousAddress, doen the rename for cotrollers as well